dev-docs/modules/userid-submodules/euid.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: userid
+title: European Unified ID
+description: European Unified ID User ID sub-module
+useridmodule: euidIdSystem
+---
+
+European Unified ID (EUID) is an open-source identity solution built for the open internet and based on the Unified ID 2.0 solution. It leverages encrypted email data to provide a privacy-conscious, secure, and accurate identity standard designed to meet market requirements in Europe and the UK.
+
+EUID relies on user consent before an ID can be added to the bid stream. Consent can be gathered by SSO providers who have integrated against the EUID framework, or through a publisher's own login and consent mechanism.
```

Comment:
would be helpful to contrast the functional differences? if seeking consent is the only difference, why not simply make one module that doesnt do anything in Europe without consent, as most id modules behave?

Reply:
@lionell-pack-ttd can you elaborate on this?

From our experience there's also another integration via LiveRamp that does this as well. So this could be interesting for readers whats the difference between Unified ID 2.0 and EUID (other than consent)